examples/resources/hpegl_metal_host/README.md

```diff
@@ -36,6 +36,7 @@ The following arguments are supported:
 - `size` - The machine size to use for this host.
 - `networks` - A list of network names or IDs on which this host will be connected and be allocated an IP address.
 - `network_route` - Network selected for the default route.
+- `network_untagged` - Network selected to be untagged.
```

Comment:
I know you are following the existing pattern, but should this be "Name of network selected to be untagged"? A network has many properties and this field is looking for the Name of that object.

Reply:
The field can be the name or the id of the network

Reply:
Let's clarify then - "Name or ID of network selected to be untagged."  Same thing for "network_route".